tests/views/test_message.py

```diff
@@ -577,3 +577,82 @@ def test_get_to_id_index_error(self):
             conversation = json.load(fp)
         del conversation['messages'][0]['msg_to'][0]
         self.assertEqual(None, _get_to_id(conversation['messages'][0]))
+
+    @requests_mock.mock()
+    @patch('response_operations_ui.controllers.message_controllers._get_jwt')
+    def test_get_close_conversation_speedbump(self, mock_request, mock_get_jwt):
+        with self.app.session_transaction() as session:
+            session['messages_survey_selection'] = 'QBS'
+        mock_get_jwt.return_value = "blah"
+        mock_request.get(url_get_thread, json=thread_json)
+        mock_request.get(url_get_surveys_list, json=survey_list)
+
+        response = self.app.get("/messages/threads/fb0e79bd-e132-4f4f-a7fd-5e8c6b41b9af/close-conversation",
+                                follow_redirects=True)
+
+        self.assertEqual(200, response.status_code)
+        self.assertIn("Subject".encode(), response.data)
+        self.assertIn("Business".encode(), response.data)
+        self.assertIn("Reference".encode(), response.data)
+        self.assertIn("Respondent".encode(), response.data)
+
+    @requests_mock.mock()
+    @patch('response_operations_ui.controllers.message_controllers._get_jwt')
+    def test_post_close_conversation_add_closed_status(self, mock_request, mock_get_jwt):
```

Comment:
I think we should rename this to `test_close_conversation`. The closed status isn't part of response ops - it's part of the api implementation - so we should avoid it becoming part of the repsonse ops tests.